docs/Automation.md

```diff
@@ -1,183 +1,198 @@
-    be used for the lifetime of the automation session unless they are
-    overridden by a Tier 1 parameter.
--   Tier 1 parameters can be passed to some commands. Tier 1 parameters are
-    never overridden.
-
-#### Return Objects
-Each command will return a .NET object to summarize the result of the operation.
-These objects are detailed under each command:
-
-### Command Details
-#### Start Command
-The Start command initializes the AxeWindows engine for automation.
-It is required for the session to begin, and each session requires a
-corresponding call to the Stop command. A given test process is allowed only one
-active session at a time, and subsequent calls to Start will fail.
-
-##### Inputs
-
-###### .NET
-
-The **StartCommand.Execute** method accepts the following parameters:
+        // Ready to use config
+        var myConfig = myConfigBuilder.build();
+
+2. Create a `Scanner` object using the `ScannerFactory` object with the `Config`.
+
+        // Create scanner using myConfig
+        var scanner = ScannerFactory.CreateScanner(myConfig);
+
+3. Call  the `Scan` method on the `Scanner` object.
+
+        var scanResults;
+        try
+        {
+            scanResults = scanner.Scan();
+        }
+        catch(AxeWindowsAutomationException e)
+        {
+            // Get the message from an exception, if one is thrown.
+            var errorMessage = e.InnerException != null ? e.InnerException.Message : e.Message;
+        }
+
+4. Check the results.
+
+        Console.WriteLine("Number of errors found in scan: " + scanResults.errorCount);
+
+
+A [complete code example](#example) can be found below.
+
+### Class Details
+
+#### Config.Builder
+
+##### *`static`* `ForProcessId`
+Create the builder for the config for the specified process.
+
+###### Parameters
+
+The `ForProcessId` method accepts the following parameters:
 
 **Name** | **Type** | **Description**
 ---|---|---
-primaryConfig | `Dictionary<string, string>` | The set of Tier 2 parameters to apply during this session. See [Defined Constants](#defined-constants) for specific parameters.
-configFile | `string` | The full path to a file that contains JSON-serialized Tier 3 parameters. Can be `string.Empty`. Must point to a valid JSON-serialized file if non-empty.
+processId | `int` | The process Id of the application to test. If the value is invalid, the automation session will throw an [`AxeWindowsAutomationException`](#error-handling).
 
-###### PowerShell
+###### Return object
 
-The **Start-AxeWindows** cmdlet accepts the following optional
-parameters:
+The `ForProcessId` method returns an instance of `Config.Builder`.
 
-**Name** | **Type** | **Description**
+##### `WithOutputFileFormat`
+
+Specify the type(s) of output files you wish AxeWindows to create.
+
+The [`OutputFileFormat` enum](../src/Automation/enums/OutputFileFormat.cs) currently has the following possible values:
+
+**Name** | **Value** | **Description**
```

Comment:
>**Value** [](start = 11, length = 9)

Not a fan of including the Value column. The user code should always be in terms of the enum, and anyone who wants to know the value can quickly determine what it is.